### ISSUE TITLE
Errors in the svr svc example

### ISSUE THREAD

**Issue by andrewczgithub** [uqfoundation/mystic#126]
Hi @mmckerns !
Hope your well.

Ther are some errors in the SVR SVC examples. in example folder 3.
```python
hyperopt.py             README           test_sparsity.py
(tutorial-env11) andrewcz-pc:[andrewcz]:~/mystic/examples3$ python test_svc1.py 
Traceback (most recent call last):
  File "test_svc1.py", line 73, in <module>
    solver.Solve(objective, ExtraArgs=(Q,b), disp=1)
  File "/home/andrewcz/tutorial-env11/lib/python3.7/site-packages/mystic-0.3.6.dev0-py3.7.egg/mystic/differential_evolution.py", line 348, in Solve
    ExtraArgs, **kwds)
  File "/home/andrewcz/tutorial-env11/lib/python3.7/site-packages/mystic-0.3.6.dev0-py3.7.egg/mystic/abstract_solver.py", line 937, in Solve
    self._Solve(cost, ExtraArgs, **settings)
  File "/home/andrewcz/tutorial-env11/lib/python3.7/site-packages/mystic-0.3.6.dev0-py3.7.egg/mystic/abstract_solver.py", line 880, in _Solve
    stop = self.Step(**settings) #XXX: remove need to pass settings?
  File "/home/andrewcz/tutorial-env11/lib/python3.7/site-packages/mystic-0.3.6.dev0-py3.7.egg/mystic/abstract_solver.py", line 853, in Step
    self._Step(**kwds) #FIXME: not all kwds are given in __doc__
  File "/home/andrewcz/tutorial-env11/lib/python3.7/site-packages/mystic-0.3.6.dev0-py3.7.egg/mystic/differential_evolution.py", line 270, in _Step
    self.trialSolution[:] = self._constraints(self.trialSolution)
  File "/home/andrewcz/tutorial-env11/lib/python3.7/site-packages/mystic-0.3.6.dev0-py3.7.egg/mystic/tools.py", line 690, in func
    return f(suppress(x, tol, clip), *args, **kwds)
  File "test_svc1.py", line 56, in conserve
    return constrain(x)
  File "/home/andrewcz/tutorial-env11/lib/python3.7/site-packages/mystic-0.3.6.dev0-py3.7.egg/mystic/coupler.py", line 75, in func
    return f(inner(x, *args, **kwds), *argz, **kwdz)
  File "<string>", line 4, in solver_140112692666000
  File "<string>", line 1
SyntaxError: can't assign to operator
(tutorial-env11) andrewcz-pc:[andrewcz]:~/mystic/examples3$ ls

```

**Comment by mmckerns** [comment]
It turns out that this error is caused by the symbolic solver failing for python 3.x.  So, this will be fixed by this issue #127.  Thus, I'm going to close this issue as a duplicate.

```
>>> from mystic.symbolic import solve
>>> c = '1.0*x0 + 1.0*x1 + 1.0*x2 + 1.0*x3 + -1.0*x4 + -1.0*x5 + -1.0*x6 + -1.0*x7 + -1.0*x8 == 0.0'
>>> solve(c)
'1.0*x0 + 1.0*x1 + 1.0*x2 + 1.0*x3 + -1.0*x4 + -1.0*x5 + -1.0*x6 + -1.0*x7 + -1.0*x8 == 0.0'
>>> c = 'x0 + 1.0*x1 + 1.0*x2 + 1.0*x3 + -1.0*x4 + -1.0*x5 + -1.0*x6 + -1.0*x7 + -1.0*x8 == 0.0'
>>>
>>> c = '1.0*x0 == -1.0*x1 - 1.0*x2 - 1.0*x3 + 1.0*x4 + 1.0*x5 + 1.0*x6 + 1.0*x7 + 1.0*x8'
>>> solve(c, target='x0')
'x0 = -1.0*x1 - 1.0*x2 - 1.0*x3 + 1.0*x4 + 1.0*x5 + 1.0*x6 + 1.0*x7 + 1.0*x8'
```

**Comment by mmckerns** [comment]
Thanks for reporting.

**Comment by mmckerns** [comment]
Wow... madness... replacing `0.0` with `0` works.

**Comment by mmckerns** [comment]
Fixed: c3a61173f1f78a20c66f10876d378558c14bbed2
